macros/calendar_date/string_to_date.sql

```diff
@@ -0,0 +1,54 @@
+{# 
+    currently supports dates stored like 'Apr 1, 2022'
+    other formats to be covered: 
```

Comment:
Does that mean these other formats are supported? Not sure I follow.

Reply:
these were just formats i hadn't written code for yet!